src/main/java/fi/otavanopisto/restfulptv/server/PtvTranslator.java

```diff
@@ -464,9 +466,25 @@ public Service translateService(IVmOpenApiService ptvService) {
     result.setChargeType(ptvService.getServiceChargeType());
     result.setAdditionalInformations(translateLocalizedListItems(ptvService.getServiceAdditionalInformations()));
     result.setTargetGroups(translateFintoItems(ptvService.getTargetGroups()));
-
+    result.setOrganizationIds(getOrganizationIds(ptvService.getOrganizations()));
+    
     return result;
   }
+  
+  private List<String> getOrganizationIds(List<VmOpenApiServiceOrganization> organizations) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getOrganizationIds" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)